src/components/editor/EditAuthorList.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import _ from 'lodash';
+import { shallowEquals } from 'lib/BaseComponent';
+
+export default class EditAuthorList extends React.Component {
```

Comment:
It might be a good idea to extend BaseComponent everywhere. It mostly adds this safeSet thing which might be useful during isomorphic rendering, but it also increases efficiency of renders, which is not as important on editor components as on readers but why not.